seminars/templates/talk.html

```diff
@@ -26,6 +26,10 @@ <h1>{{talk.show_title()}}{% if not talk.display %} (hidden){% elif talk.online %
   <p>
     {{ talk.show_time_and_duration() | safe }}{% if talk.timezone != current_user.timezone %} in your time zone, {{ current_user.timezone.replace("_", " ") }}.{% endif %}
   </p>
+  {% if not talk.online and talk.timezone != current_user.timezone %}
```

Comment:
Is it better for this to be "not talk.online" or "talk.room"?  I'm fine with the way you've written it, but I think the plan is for the presence of a room to indicate an offline presence, which is maybe what you're looking for.